app/src/main/java/com/twilio/notify/quickstart/notifyapi/TwilioSDKStarterAPI.java

```diff
@@ -9,7 +12,7 @@
 import retrofit2.http.POST;
 
 public class TwilioSDKStarterAPI {
-    public final static String BASE_SERVER_URL = "YOUR_SDK_STARTER_SERVER_URL";
+    public final static String BASE_SERVER_URL = "https://f107bbc9.ngrok.io";
```

Comment:
Please change this back to "YOUR_SDK_STARTER_SERVER_URL" so users know to change to their own local ngrok/server